go-controller/pkg/libovsdbops/router.go

```diff
@@ -975,8 +997,9 @@ func DeleteNATsWithPredicateOps(nbClient libovsdbclient.Client, ops []libovsdb.O
 func CreateOrUpdateGatewayChassis(nbClient libovsdbclient.Client, port *nbdb.LogicalRouterPort, chassis *nbdb.GatewayChassis) error {
 	opModels := []operationModel{
 		{
-			Model:          chassis,
-			OnModelUpdates: onModelUpdatesAll(),
+			Model: chassis,
+			// TODO(trozet): evaluate if only these fields should be updated
+			OnModelUpdates: []interface{}{&chassis.ChassisName, &chassis.Priority},
```

Comment:
I think this is missing the name.

With that, it is equivalent as what we had before as those are the values that we pass.

Reply:
done